code/game/gamemodes/blob/theblob.dm

```diff
@@ -121,12 +121,15 @@
 			else	T = null
 
 	if(!T)	return 0
-	var/obj/effect/blob/normal/B = new /obj/effect/blob/normal(src.loc, min(src.health, 30))
+	var/obj/effect/blob/B = new /obj/effect/blob/normal(src.loc)
+	if(istype(T, /turf/space) && prob(65))
+		B.health = 0
 	B.color = a_color
 	B.density = 1
 	if(T.Enter(B,src))//Attempt to move into the tile
 		B.density = initial(B.density)
 		B.loc = T
+		B.update_icon()

```

Comment:
Why do that change? What's bad about just doing `if(istype(T, /turf/space) && prob(65)) 	return` ?